app/views/lead_providers/your_schools/index.html.erb

```diff
@@ -18,10 +18,13 @@
   </div>
 </div>
 
-<%= render SubnavComponent.new do |component| %>
-  <% @cohorts.each do |cohort| %>
-    <%= component.nav_item(path: lead_providers_your_schools_path(selected_cohort_id: cohort.id)) do %>
-      <%= cohort.start_year %> cohort
+<% if @cohorts.length > 1 %>
+  <%= render SubnavComponent.new do |component| %>
+    <% @cohorts.each do |cohort| %>
+      <%# The link to the current cohort doesn't have it in the URL %>
+      <%= component.nav_item(path: lead_providers_your_schools_path(cohort: cohort.start_year === Time.zone.today.year ? nil : cohort)) do %>
```

Comment:
One day I will add something that crashes the server if it sees `===` outside of js. Until that day, I will simply comment saying that this is not js land, we should use `==` 😂

Reply:
:lolsob: